MAINTAINERS.md

```diff
@@ -0,0 +1,197 @@
+# Maintainers
+
+This document serves to clarify the responsibilities of a maintainer as well as
+serve as a wiki for maintainers to link to / include resources.
+
+
+## Current Maintainers
+
+The current maintainers are:
+
+| name | focus areas |
```

Comment:
I think as long as maintainers know what other focus other maintainers have currently, they can point people in the right direction. i.e., I don't think it's required for this PR

Reply:
this is made clear in the following part of the document:


> The role of the Maintainer is currently evolving with respect to "Scoped Maintainers". The concept of a scoped maintainer is that it is helpful to add maintainers who are subject matter experts in a particular area of the code, but perhaps not in other areas. As such, maintainers with an explicit scope typically will avoid merging PRs in areas outside their scope, but may do so on occasion. As an individual maintainer's experience levels in the project increase, they may change their focus from scoped to general, but this is not currently an explicit designation (all maintainers should be considered to be general at this time). As a rough measure, scoped maintainers should avoid "stepping on the toes" of work that is being guided by another maintainer or contributor.


I'm not sure if the "review bar" for updating documentation is the same as for modifying code, but I would assume it would be OK for maintainers to self-merge if they are keeping it up to date. As the document describes, this is just documenting what a maintainer's focus is, not restricting it.